environment.yml

```
@@ -28,8 +28,8 @@ dependencies:
   - flask-cors==4.0.0
   - redis-py==5.0.1
   - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
+  - Scrapy==2.11.1
```

```suggestion
  - scrapy==2.11.1
```
@Flame1190 I'd suggest changing that to small letter s, not sure if capital letter is not.